lib/taskana-core/src/main/java/pro/taskana/monitor/internal/reports/TimestampReportBuilderImpl.java

```diff
@@ -69,7 +69,7 @@ public TimestampReport buildReport() throws NotAuthorizedException, InvalidArgum
           status.stream()
               // This can also be implemented into a single sql query which combines all statuses
               // with the union operator. That would reduce the readability of the sql template.
-              // That's why "the loop" is done outside of mybatis.
+              // That's why "the loop" is done outside mybatis.
```

Comment:
> I don't wanna make "mimimi", but

 IntelliJ made this suggestion, because the `of` is unnecessary here
and i wasn't sure so i googled it and the `of` is usually left out in BE and always left in in AE
so both are valid, but IntelliJ preferred this one